src/_includes/partials/author_bios/haydenbleasel.md

```diff
@@ -0,0 +1,3 @@
+Part designer, part developer. Usually found running Design and/or Product at startups and tech companies.
```

Comment:
Do you want to expand on this? We usually do these bios in the third person, such as “Hayden Bleasel is the head of product and design at Corellium, where he….In his spare time he also…” or whatever else seems helpful to you. 3-4 sentences would be great.

Reply:
Yeah absolutely, wasn't sure how much to write but I'll revisit now.